CE07SHSM/CE07SHSM_R00018_ingest.csv

```diff
@@ -0,0 +1,54 @@
+mi.dataset.driver.presf_abc.presf_abc_wave_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl36/PRESF*/*presf*.hex,CE07SHSM-MFD35-02-PRESFB000,recovered_inst,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl36/adcpt2/*.adcpt*.log,CE07SHSM-MFD35-04-ADCPTC000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl36/ADCPT*/*.000,CE07SHSM-MFD35-04-ADCPTC000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl36/pco2w/*.pco2w.log,CE07SHSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl36/PCO2W*/*pco2w*recovered*.txt,CE07SHSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl36/phsen2/*.phsen*.log,CE07SHSM-MFD35-06-PHSEND000,recovered_host,Available,
+# mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl36/PHSEN*/*phsen*recovered*.txt,CE07SHSM-MFD35-06-PHSEND000,recovered_inst,Not Expected,Instrument failed to return data
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl37/syslog/*.syslog.log,CE07SHSM-MFD37-00-DCLENG000,recovered_host,Pending,Incorrectly specified dataset and parser
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl37/optaa2/*.optaa*.log,CE07SHSM-MFD37-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl37/ctdbp2/*.ctdbp*.log,CE07SHSM-MFD37-03-CTDBPC000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl37/CTDBP*/*ctdbp*.hex,CE07SHSM-MFD37-03-CTDBPC000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl37/ctdbp2/*.ctdbp*.log,CE07SHSM-MFD37-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp.dosta_abcdjm_ctdbp_recovered_driver,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl37/CTDBP*/*ctdbp*.hex,CE07SHSM-MFD37-03-DOSTAD000,recovered_inst,Available,
+# ,/omc_data/whoi/OMC/CE07SHSM/R00018/cg_data/dcl37/camds/*.png,CE07SHSM-MFD37-06-CAMDSA000,recovered_host,Pending,No parser exists for this data set
+# ,/omc_data/whoi/OMC/CE07SHSM/R00018/instrmts/dcl37/CAMDS_*/*camds*/DCIM/*.png,CE07SHSM-MFD37-06-CAMDSA000,recovered_inst,Pending,No parser exists for this data set

```

Comment:
Again, I know this is commented out, but there is no DCIM directory under the CAMDS instrmnts directory, only {jpg,logs,raw}. Not sure if this will ever become relevant someday or not?

Reply:
Ignoring the CAMDS for now. At some point, I'll either delete those entries entirely, or update accordingly.